examples/express-advanced-server.js

```diff
@@ -47,6 +47,7 @@ gracefulShutdown(server,
     development: false,
     onShutdown: cleanup,
     forceExit: true,
+    shutdownTimeout: 5000,
```

Comment:
suggestion: for the upstream library it might make sense to extend this differently. instead of the concrete use-case of "wait this much time after i receive a signal", it would be more extensible if we had an `onSignal` or `beforeCleanup` function which runs whatever code is passed into it before we close connections.